lib/utils/domFns.js

```diff
@@ -109,13 +109,23 @@ export function offsetXYFromParent(evt: {clientX: number, clientY: number}, offs
   return {x, y};
 }
 
-export function createCSSTransform({x, y}: {x: number, y: number}): Object {
+export function createCSSTransform({x, y}: {x: number, y: number}, defaultPosition: {x: number | string, y: number | string }): Object {
+  const defaultX = (typeof defaultPosition.x === 'string') ? defaultPosition.x : defaultPosition.x + 'px';
```

Comment:
This is really difficult to read. You create these two consts, `defaultX` and `defaultY`, but they're only actually used if and only if a `defaultPosition` is actually present. 

You then use that just to prepend what is being returned as a second translate. 

This is a lot of logic in what was previously a simple function and should be refactored such that the caller prepends the default translate if needed.

Reply:
Reworked this to hopefully be more clear